pkg/access_token/access_token_file_test.go

```diff
@@ -19,138 +19,223 @@ func TestAccessTokenFile(t *testing.T) {
 	var tokenInFile, _ = NewAccessTokenFile(config)
 
 	Convey("Read", t, func() {
-		Convey("Returns true if Data output as expected", func() {
+
+		Convey("Given an access token with data", func() {
 			dataActual := []byte{'t', 'e', 's', 't'}
 			err := tokenInFile.Write(dataActual)
-			So(err, ShouldEqual, nil)
 
-			dataExpected, err := tokenInFile.Read()
-			eq := reflect.DeepEqual(dataActual, dataExpected)
+			Convey("Finishes without raising an error", func() {
```

Comment:
this is actually part of the given clause of this test. can we change it to `That finishes without raising an error`?

Reply:
we should make clear what we test in each tet

Reply:
I am not sure I understand here. Make it part of the `given` or the 'it' is implicit clause?

Reply:
Given. it's not what you're actually testing here. it is given that it finishes without raising an error *and only then* you start the actual test